README.md

```diff
@@ -404,7 +410,7 @@ NOTE: VHL uses the less cutting-edge EcmaScript 5 version of this Guide!
     const name = 'Capt. Janeway';
     ```
 
-  - [6.2](#6.2) <a name='6.2'></a> Strings longer than 80 characters should be written across multiple lines using string concatenation.
+  - [6.2](#6.2) <a name='6.2'></a> Strings longer than 100 characters should be written across multiple lines using string concatenation.
```

Comment:
I will make any of our own alterations/customizations in a separate branch.  Thanks for pointing this out.  I do NOT want to start the whole 80 vs 100 vs 120 thing again.